src/tools/tidy/src/error_codes.rs

```diff
@@ -95,7 +95,6 @@ fn check_removed_error_code_explanation(ci_info: &crate::CiInfo, bad: &mut bool)
         eprintln!("Take a look at E0001 to see how to handle it.");
         return;
     }
-    println!("No error code explanation was removed!");
```

Comment:
These messages are useful, especially in regression checks later on if this code because outdated (ie, how error codes are handled changed).

Reply:
I really don't think we should print messages of the form "there was no error in XYZ". That's about as user-friendly as rustc printing
```
No type error found.
No lifetime error found.
No violated trait bound found.
...
```

Testing-only messages useful for debugging the tool should not be enabled by default in production.